frame/evm/src/lib.rs

```diff
@@ -133,14 +212,19 @@ static ISTANBUL_CONFIG: Config = Config::istanbul();
 
 /// EVM module trait
 pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
-	/// The EVM's module id
-	type ModuleId: Get<ModuleId>;
 	/// Calculator for current gas price.
 	type FeeCalculator: FeeCalculator;
-	/// Convert account ID to H160;
-	type ConvertAccountId: ConvertAccountId<Self::AccountId>;
-	/// Currency type for deposit and withdraw.
+
+	/// Allow the origin to call on behalf of given address.
+	type CallOrigin: EnsureAddressOrigin<Self::Origin>;
```

Comment:
Learning question: What is the type success when you don't specify it in this case ?

Reply:
We cannot not specify this -- it would otherwise be a compile error. So the runtime declaration has to be changed now to contain the new type `CallOrigin`.